### PR TITLE
Fix to start when there is no config file

### PR DESCRIPTION
**Pull request by ohkinozomu**
resolve https://github.com/containerd/stargz-snapshotter/issues/13

Signed-off-by: Nozomu Ohki <nozomunoise@gmail.com>

### CODE REVIEW

File: script/integration/containerd/entrypoint.sh [hunk 1]
```diff
@@ -247,4 +250,8 @@ diff --no-dereference -qr "${USR_NORMALSN_PLAIN_STARGZ}/" "${USR_STARGZSN_PLAIN_
 ctr-remote --namespace=dummy images rpull --user "${DUMMYUSER}:${DUMMYPASS}" \
            "${REGISTRY_HOST}:5000/ubuntu:esgz"
 
+############
+# Test for starting when no configuration file.
+reboot_containerd nonexistent.toml
```
Comment (AkihiroSuda) [inline script/integration/containerd/entrypoint.sh:255]: When a non-existent path was explicitly specified, it should fail

Comment (ohkinozomu) [reply script/integration/containerd/entrypoint.sh]: I misunderstood.
https://github.com/containerd/stargz-snapshotter/pull/229/commits/7d99fc8b3efc038f7bb27e031f601230caecac98

**Review by AkihiroSuda — APPROVED** [review]
LGTM, but please squash commits

**Comment by ohkinozomu** [comment]
squashed

**Review by AkihiroSuda — APPROVED** [review]

**Review by ktock — APPROVED** [review]
Thanks!

**Pull request merged**